ivy_tests/test_ivy/test_frontends/test_torch/test_reduction_ops.py

```diff
@@ -949,3 +949,33 @@ def test_torch_unique_consecutive(
         dim=axis,
         test_values=False,
     )
+
+@handle_frontend_test(

```

Comment:
Since it's an instance method, the test should be written in `test_torch/test_tensor.py` and you should use `@handle_frontend_method` decorator for writing the test. Please read the Frontend Instance Method Testing section in the following link: https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html#frontend-instance-method-tests